### PR TITLE
[fix](array-type) fix the wrong data when use stream load to import '\N'

### PR DESCRIPTION
**Pull request by carlvinhust2012**
# Proposed changes
1. this pr is used to fix the wrong data when use stream load to import '\N';
2. before the fix , we import the '\N' will encounter the wrong data:
the origin data is:
1       [1, 2, 3]
2       [4, 5, 6]
3       \N

MySQL [example_db]> select * from array_test06;
+------+-----------+
| k1   | k2        |
+------+-----------+
|    1 | [1, 2, 3] |
|    2 | [4, 5, 6] |
|    3 | [4, 5, 6] |
+------+-----------+
9 rows in set (0.008 sec)

3. after the fix, the data will correct:
MySQL [example_db]> select * from array_test06;
+------+-----------+
| k1   | k2        |
+------+-----------+
|    1 | [1, 2, 3] |
|    2 | [4, 5, 6] |
|    3 | NULL      |
+------+-----------+
3 rows in set (0.011 sec)
 
Issue Number: #7570

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
5. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
6. Has document been added or modified:
    - [ ] Yes
    - [x] No
    - [ ] No Need
7. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
8. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by adonis0147 — DISMISSED** [review]
LGTM

File: regression-test/data/load_p0/broker_load/simple_array.data [hunk 1]
```diff
@@ -0,0 +1,3 @@
+1/[1,2,3,4,5]/[32767,32768,32769]/[65534,65535,65536]/["a","b","c","d","e"]/["hello","world"]/["1991-01-01"]/["1991-01-01 00:00:00"]/[0.33,0.67]/[3.1415926,0.878787878]/[1,1.2,1.3]
+2/[1,2,3,4,5]/[32767,32768,32769]/[65534,65535,65536]/["a","b","c","d","e"]/["hello","world"]/\N/\N/\N/\N/[1,1.2,1.3]
+3/\N/\N/\N/\N/\N/\N/\N/\N/\N/\N
```
Comment (adonis0147) [inline regression-test/data/load_p0/broker_load/simple_array.data:3]: I think the cases should cover partial `\N`s in one array. E.g. [1, \N, 3]

**Review by adonis0147 — CHANGES_REQUESTED** [review]

File: regression-test/data/load_p0/broker_load/simple_array.data (hunk 1 above)
Comment (carlvinhust2012) [reply regression-test/data/load_p0/broker_load/simple_array.data:3]: done

**Review by adonis0147 — APPROVED** [review]
LGTM

**Pull request merged**